src/transformers/tokenization_transfo_xl.py

```diff
@@ -286,7 +303,9 @@ def save_vocabulary(self, vocab_path):
             vocab_file = os.path.join(vocab_path, VOCAB_FILES_NAMES["pretrained_vocab_file"])
         else:
             vocab_file = vocab_path
-        torch.save(self.__dict__, vocab_file)
+        with open(vocab_file, "wb") as f:
+            pickle.dump(self.__dict__, f)
+        # torch.save(self.__dict__, vocab_file)
```

Comment:
This line doesn't need to be there anymore now, does it?

Reply:
Yes, this shouldn't be needed anymore, it was there temporarily to make the point ;)